tests/physics/resolve/ball_ball/test_ball_ball.py

```diff
@@ -111,3 +111,82 @@ def test_head_on_z_spin(model: BallBallCollisionStrategy, cb_wz_i: float):
     cb_wz_f = cb_f.state.rvw[2][2]
     assert cb_wz_f > 0, "Spin direction shouldn't reverse"
     assert cb_wz_f < cb_wz_i, "Spin should be decay"
+
+
+@pytest.mark.parametrize(
+    "model", [FrictionalInelastic(), FrictionalMathavan(num_iterations=int(1e5))]
```

Comment:
Mathavan takes this many iterations to meet requirements for no throw and no induced english.